lib/matplotlib/legend.py

```diff
@@ -294,6 +294,7 @@ class Legend(Artist):
     def __str__(self):
         return "Legend"
 
+    @_api.rename_parameter("3.6", "ncol", "ncols")
```

Comment:
This is _very_ old and pretty heavily used.  I am not sure we should deprecate it?  Can we just alias it and change the docs to the preferred grammar?

Reply:
Sure!

Reply:
Should I even make a release note then? Or just let (new) users slowly realize that `ncols` is the preferred name?

Reply:
I think a what's new is helpful....